### PR TITLE
Feature/tomlinson/sign up page

### PR DESCRIPTION
**Pull request by mark-tomlinson-dev**
## Details
- Added sign up and log in pages
- Created secondary navigation header
- Created basic button styles

### CODE REVIEW

File: client/src/components/Header.js [hunk 1]
```diff
@@ -59,8 +59,10 @@ const Header = (props) => {
               </div>
               <div className={styles.navigation}>
                 <nav>
-                  <button onClick={handleTogglePopUpOnLogin}>Log in</button>
-                  <button onClick={handleTogglePopUpOnSignUp}>Sign up</button>
+                  {/* <button onClick={handleTogglePopUpOnLogin}>Log in</button>
+                  <button onClick={handleTogglePopUpOnSignUp}>Sign up</button> */}
```
Comment (maxckelly) [inline client/src/components/Header.js:63]: Should we remove the commented code?

File: client/src/components/LoginForm.js [hunk 2]
```diff
@@ -84,12 +87,12 @@ let LoginForm = props => {
...
           />
 
-          {errors.password && errors.password.types.required && (<p>Password required</p>)}
+          {errors.password && errors.password.types.required && (<h5>Password required</h5>)}
         </div>
 
         <div className={styles.buttons}>
-          <button onClick={handlePopUpClose}> Cancel </button>
-          <button type="submit"> Submit </button>
+          {/* <button onClick={handlePopUpClose}> Cancel </button> */}
```
Comment (maxckelly) [inline client/src/components/LoginForm.js:94]: Let's remove this commented code

**Comment by maxckelly** [comment]
We should probably get rid of the function below because we aren't handling the popup just yet. 

`  const handlePopUpClose = () => {
    props.togglePopUpOffSignUp()
  }`

File: client/src/components/Header.js (hunk 1 above)
Comment (mark-tomlinson-dev) [reply client/src/components/Header.js:80]: Yep will remove now

**Pull request merged**